src/public/client.ts

```diff
@@ -41,15 +41,18 @@ class MappingItem {
     const mappingElement = document.createElement('li')
     let iconClass
     let iconColor
+    let logClass
```

Comment:
`let logClass = 'fa fa-file-text-o ml-1 mt-1'`

Reply:
why does this even have to be a variable? and putting this inside the `status` check block is kinda strange, what does status have to do with logs?

Reply:
This to disable UI for getting logs when pm2 doesn't return a status. (i.e for c0d3.com's application)